flang/lib/Lower/OpenMP/DataSharingProcessor.cpp

```diff
@@ -135,138 +133,135 @@ void DataSharingProcessor::insertBarrier() {
 void DataSharingProcessor::insertLastPrivateCompare(mlir::Operation *op) {
   bool cmpCreated = false;
   mlir::OpBuilder::InsertPoint localInsPt = firOpBuilder.saveInsertionPoint();
-  for (const Fortran::parser::OmpClause &clause : opClauseList.v) {
```

Comment:
Are there code changes here or is it only  the replacement of `parser::OmpClause` with `omp::Clause`?

Reply:
The code functionality is unchanged.  One reason for the seemingly large differences in the diff is that I replaced the `if (get_if<LastPrivate>)` with `if (!get_if<Lastprivate>) continue;` to reduce indentation.